### ISSUE TITLE
view 3: form to request a specific model calculation

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#9]
initially likely a request for on-demand scenario simulation for the use of professionals / decision makers / health pros; the user would have the option to specify more precise location, and we would initially run the required simulations by hand [source of description](https://epidemicforecasting.slack.com/archives/CV1F9NB5K/p1583621895004500). 

A user should be able to submit a form, which is somehow persisted (either via [sending it to an email](https://formspree.io/) or to a DB).

This will apply to the content reachable via navigation menu "Request Model" in the website

So basically a form which would submit some stuff in our DB

Parameters ([source of requirements](https://epidemicforecasting.slack.com/archives/CV1F9NB5K/p1583872152041500?thread_ts=1583621895.004500&cid=CV1F9NB5K)): 
- Location
- Use case; 

If there are some typical use cases, we could automate it
Bonus points for knowing how to automate the recalculation of the model in the future.

## AC
* the form is available in the site and one can submit the parameters which are persisted

**Comment by Bachibouzouk** [comment]
@hnykda - it would have been easier to say "This should look like'Private Form Copy'" in https://www.figma.com/file/4FTT6neJVlMD5CQHTTpprF/Covid

**Comment by hnykda** [comment]
These designs arrived after this was specced :-)
